etna/ciim/client.py

```diff
@@ -171,8 +171,8 @@ def fetch(
 
         Keyword arguments:
 
-        iaid:
-            Return match on Information Asset Identifier
+        metadata_id:
+            Return match on @admin.id for various Information Asset Identifier (IAID) formats.
         id:
             Generic identifier. Matches on references_number or iaid
```

Comment:
Noted this - perhaps a discussion will be required

Reply:
@ababic , it requires availability of those concerned on these points. I have raised them and can get back - but it can be over next week if you want to hold it  at all.